Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -432,7 +432,7 @@ static IntSize computeMaximumSurfaceSize()
 {
     auto configuration = bitmapConfiguration();
     auto bitsPerPixel = configuration.bitsPerComponent * 4;
-    auto bytesPerRow = width * bitsPerPixel;
+    auto bytesPerRow = roundUpToMultipleOfNonPowerOfTwo(width * (bitsPerPixel / 8), bytesPerRowAlignment());
```

Comment:
I think these parameters are in the wrong order, first param is the divisor.

Reply:
Oh yes that is certainly right, thanks for catching, updated PR correcting this